### PR TITLE
docs: Adjust the TF override code documentation

### PR DESCRIPTION
**Pull request by jeromelaban**

### CODE REVIEW

**Review by agneszitte-nventive — COMMENTED** [review]
@jeromelaban maybe we can also make the steps clearer here at the same time?
image
https://platform.uno/docs/articles/uno-development/building-uno-ui.html

**Comment by amelield** [comment]
This is really easy to understand with that tab. I really like it.

**Comment by jeromelaban** [comment]
> @jeromelaban maybe we can also make the steps clearer here at the same time?

How would you phrase it?

**Comment by amelield** [comment]
Maybe at step 5, after Save the file, we could add something like: Close the file, then open solution in Visual Studio? It would insure the file is closed when the solution is opened. Also I seem to remember seeing that the `global.json` modification at step 6 is no longer needed. Or I am mistaken?

**Comment by jeromelaban** [comment]
> Maybe at step 5, after Save the file, we could add something like: Close the file, then open solution in Visual Studio? It would insure the file is closed when the solution is opened. I even think that just a "Save the file, then close it." would do. Also I seem to remember seeing that the `global.json` modification at step 6 is no longer needed. Or I am mistaken?

Indeed, global.json should be adjusted. As for the file being closed, it's not required, it just needs to be saved on disk.

File: doc/articles/uno-development/building-uno-ui.md [hunk 1]
```diff
@@ -40,13 +40,12 @@ The step by step process is:
...
 1. In `crosstargeting_override.props`, uncomment the line `<UnoTargetFrameworkOverride>netstandard2.0</UnoTargetFrameworkOverride>`
-1. Set the build target inside ``<UnoTargetFrameworkOverride></UnoTargetFrameworkOverride>`` to the identifier for the target platform you wish to build for. (Identifiers for each platform are listed in the file.) Save the file.
+1. Set the build target inside ``<UnoTargetFrameworkOverride></UnoTargetFrameworkOverride>`` to the identifier for the target platform you wish to build for (Identifiers for each platform are listed in the `crosstargeting_override.props` file), then save the file then open the appropriate solution filter file in Visual Studio.
 1. If you are debugging for `net6.0-XX` targets (`ios`, `android`, `maccatalyst` or `macos`)
-   - Ensure that you are running VS 2022 17.1 Preview 1 or later
-   - Replace the contents of the `global.json` file with the contents of `global-net6.json`
+   - Ensure that you are running VS 2022 17.3 or later
 1. In the `src` folder, look for the solution filter (`.slnf` file) corresponding to the target platform override you've set, which will be named `Uno.UI-[Platform]-only.slnf`, and open it.
```
Comment (agneszitte-nventive) [inline doc/articles/uno-development/building-uno-ui.md:49]: @jeromelaban this step should be right above this step no?
> Set the build target inside ``<UnoTargetFrameworkOverride></UnoTargetFrameworkOverride>`` to the identifier for the target platform you wish to build for (Identifiers for each platform are listed in the `crosstargeting_override.props` file), **then save the file then open the appropriate solution filter file in Visual Studio.**

File: doc/articles/uno-development/building-uno-ui.md [hunk 2]
```diff
@@ -40,13 +40,12 @@ The step by step process is:
 
 1. Clone the Uno.UI repository locally, and ensure using a short target path, e.g. _D:\uno_ etc.  
 This is due to limitations in the legacy .NET versions used by Xamarin projects. This issue has been addressed in .NET 5, and will come to the rest of the projects in the future.
-1. Make sure you don't have the Uno.UI solution open in any Visual Studio instances. (Visual Studio may crash or behave inconsistently if it's open when the target override is changed.)
+1. Make sure you don't have the Uno.UI solution opened in any Visual Studio instances. (Visual Studio may crash or behave inconsistently if it's open when the target override is changed.)
```
Comment (agneszitte-nventive) [inline doc/articles/uno-development/building-uno-ui.md:43]: @jeromelaban Maybe we can add an additional note here or at the end of the steps to specify that between each platform when changing the identifier in `crosstargeting_override.props` they need again to do the same thing, i.e. closing VS first, updating the identifier, and re-loading the proper solution filter (`.slnf` file) corresponding to the new target platform override they have set

Comment (agneszitte-nventive) [reply doc/articles/uno-development/building-uno-ui.md:43]: @jeromelaban did you have an additional note or step regarding the comment I left? Not sure I'm seeing this info in the updated doc in the PR

**Pull request merged**